packages/@react-native-windows/cli/src/runWindows/utils/autolink.ts

```diff
@@ -980,6 +1049,11 @@ export const autolinkOptions: CommandOption[] = [
       "Override the app project file determined by 'react-native config', e.g. windows\\myApp\\myApp.vcxproj",
     default: undefined,
   },
+  {
+    name: '--no-telemetry',
+    description:
```

Comment:
update the CLI readme with this new option

Reply:
Issue: https://github.com/microsoft/react-native-windows-samples/issues/602